### DISCUSSION TITLE
How to exclude a built-in library?

### DISCUSSION THREAD

**Discussion by kkinnear** [babashka/babashka#1337] [Q&A] [answered]
I've moved zprint `1.2.4` to use `rewrite-clj` version `1.1.45`, where `edn*` is deprecated.  So, I changed zprint to use `of-node*`.  Now, unfortunately, zprint won't work with babashka.  I'm presuming that is because babashka builds in an older version of `rewrite-clj`. I have encountered similar issues with zprint and `rewrite-clj` in other libraries, and more or less solved them by excluding `rewrite-clj` from the other libraries that included it.  I was going to try the same thing with babashka, but despite quite a lot of searching and trial and error, I've not found any way to exclude `rewrite-clj` from babashka.

I've love any ideas you have, either for how to exclude `rewrite-clj` from babashka, or any other way to solve this problem.

Thanks!

**Comment by borkdude** [comment, marked as answer]
@kkinnear Rewrite-clj was updated in bb master. So this issue should resolve itself automatically from the next bb release on, but it would be good to test it. 

Can you try with babashka from master?

You can download one here: https://github.com/babashka/babashka-dev-builds

**Reply by kkinnear** [reply]
That worked great, thanks.  I'll pass that along to the zprint user that prompted this question, and I'll up my testing game so that I don't let something like this slip through again.

Thanks again for the answer, and for babashka itself.
